snapcraft/cli/__init__.py

```diff
@@ -82,8 +83,9 @@ def get_command(self, ctx, cmd_name):
 
 @click.group(cls=SnapcraftGroup, invoke_without_command=True)
 @click.pass_context
+@add_build_options(hidden=True)
 @click.option('--debug', '-d', is_flag=True)
-def run(ctx, debug, catch_exceptions=False):
+def run(ctx, debug, catch_exceptions=False, **kwargs):
```

Comment:
I missed this in the previous review, but I'll mention it now. `ctx` is a bad name, `context` is a lot better.